app/e2e/packetPage.local.ts

```diff
@@ -0,0 +1,100 @@
+      await expect(await depItems.last().getByRole("link")).toHaveAttribute("href", `/${dep2Name}/${dep2Id}`);
+    });
+
+    test("can see reports", async ({ page }) => {
+      await page.goto(`/artefact-types/${artefactTypesPacketId}`);
+      const content = await getContentLocator(page);
+      const reportsDiv = await getPacketPageAccordionSection(content, "Reports");
+      const reportFrame = await reportsDiv.locator("iframe");
+      await expect(reportFrame).toBeVisible();
+      await expect(await reportFrame.contentFrame().getByRole("heading")).toHaveText("TEST");
+    });
+  });
+
+  test.describe("Metadata", () => {
+    let content: Locator;
+
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`./parameters/${parametersPacketId}`);
+      content = await getContentLocator(page);
+      await selectPacketPageTab(content, "Metadata");
+    });
+
+    test("can see timings", async () => {
+      const timingsDiv = await getPacketPageAccordionSection(content, "Timings");
+      const timingItems = await timingsDiv.getByRole("listitem");
+      await expect(await timingItems).toHaveCount(2);
+      await expect(await timingItems.first()).toHaveText("StartedThu, 29 Aug 2024 18:54:40 GMT");
+      await expect(await timingItems.last()).toHaveText("Elapsed192 milliseconds");
+    });
+
+    test("can see git", async () => {
+      const gitDiv = await getPacketPageAccordionSection(content, "Git");
+      // get top level list items
+      const listItems = await gitDiv.locator("ul.space-y-1 > li").all();
+      await expect(await listItems.length).toBe(3);
+      await expect(listItems[0]).toHaveText("Branch" + "mrc-5661-fe-run-params");
+      await expect(listItems[1]).toHaveText("Commit" + "4c2b5d37deada04981a906f9705f0414b8ea0dfc");
+      await expect(await listItems[2].locator("span")).toHaveText("Remotes");
+      const remoteItems = await listItems[2].getByRole("listitem");
+      await expect(remoteItems).toHaveCount(1);
+      await expect(remoteItems.first()).toHaveText("https://github.com/mrc-ide/packit.git");
+    });
+
+    test("can see platform", async () => {
+      const platformDiv = await getPacketPageAccordionSection(content, "Platform", true);
+      // get top level list items
+      const listItems = await platformDiv.locator("ul.space-y-1 > li").all();
+      await expect(listItems.length).toBe(3);
+      await expect(listItems[0]).toHaveText("OS" + "Ubuntu 22.04.4 LTS");
+      await expect(listItems[1]).toHaveText("Systemx" + "86_64, linux-gnu");
+      await expect(listItems[2]).toHaveText("Language" + "R version 4.4.1 (2024-06-14)");
+    });
+
+    test("can see packages", async () => {
+      const packagesDiv = await getPacketPageAccordionSection(content, "Packages", true);
+      const listItems = await packagesDiv.locator("ul.space-y-1 > li").all();
+      await expect(listItems.length).toBe(21);
+      await expect(listItems[0]).toHaveText("askpass" + "1.2.0");
+    });
+  });
```

Comment:
One of us should add some e2e tests for the downloads tab.

Reply:
One of us made a ticket at least :)
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6240/E2e-tests-for-downloads